mars/tensor/base/tests/test_base.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 1999-2018 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+from mars.tensor.base import atleast_1d, atleast_2d, atleast_3d
+from mars.tensor.expressions.base import transpose, squeeze, moveaxis, atleast_1d, atleast_2d, atleast_3d
```

Comment:
Remove the repeated import `atleast_1d`, ..., `atleast_3d` according to cadacy.

Reply:
hi, I have push the latest commit.

Reply:
Merged already